hibernate5/src/main/java/com/baeldung/hibernate/MultiTenantHibernateTest.java

```diff
@@ -0,0 +1,73 @@
+package com.baeldung.hibernate;
+import static org.junit.Assert.assertNotEquals;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.baeldung.hibernate.pojo.Suppliers;
+
+
+public class MultiTenantHibernateTest  {
+    @Test
+    public void givenDBMode_whenFetchingSuppliers_thenComparingFromDbs () {
+        SessionFactory sessionFactory;
+        try {
+            sessionFactory = HibernateMultiTenantUtil.getSessionFactory();
+            
+            Session db1Session = sessionFactory
+                .withOptions().tenantIdentifier("mydb1").openSession();
+            
+            initDb1(db1Session);
+            
+            Transaction transaction = db1Session.getTransaction();
+            transaction.begin();
+            Suppliers supplierFromDB1 = (Suppliers)db1Session.createCriteria(Suppliers.class).list().get(0);
+            transaction.commit();
+            
+            Session db2Session = sessionFactory
+                .withOptions().tenantIdentifier("mydb2").openSession();
+            
+            initDb2(db2Session);
+            db2Session.getTransaction().begin();
+            Suppliers supplierFromDB2 = (Suppliers) db2Session.createCriteria(Suppliers.class).list().get(0);
+            db2Session.getTransaction().commit();
+            
+            System.out.println(supplierFromDB1);
+            System.out.println(supplierFromDB2);
+            
+            assertNotEquals(supplierFromDB1, supplierFromDB2);   
+        } catch (UnsupportedTenancyException e) {
+            e.printStackTrace();
+        }
+    }
+
+
+
+    private void initDb1(Session db1Session) {
+        System.out.println("Init DB1");
+        Transaction transaction = db1Session.getTransaction();
+        transaction.begin();
+        db1Session.createSQLQuery("DROP ALL OBJECTS").executeUpdate();
```

Comment:
@KevinGilmore well since we are using legacy config method (SessionManager) for boostrapping against the EntityManager, we will have to use this functions though deprecated as of now for multitenancy example. As there is no documentation of how to do that with entitymanager bootstrapping method.